crates/node/src/mempool/mod.rs

```diff
@@ -108,12 +106,3 @@ impl networking::p2p::TxHandler for P2PTxHandler {
         self.0.write().await.add(tx).await
     }
 }
-
-#[async_trait::async_trait]
-impl networking::p2p::TxChannel for networking::p2p::P2P {
-    async fn send_tx(&self, tx: &Transaction) -> Result<()> {
-        let bs = Bytes::from(bincode::serialize(tx)?);
-        self.broadcast(bs)?;
-        Ok(())
-    }
-}
```

Comment:
I have zero idea why I had put this here but it's in the wrong place :see_no_evil: